src/components/backend-ai-user-list.ts

```diff
@@ -489,7 +482,7 @@ export default class BackendAIUserList extends BackendAIPage {
     const confirmEl = this.shadowRoot.querySelector('#confirm');
     const confirm = confirmEl.value;
     const description = this.shadowRoot.querySelector('#description').value;
-    const is_active = this.shadowRoot.querySelector('#is_active').selected;
+    const status = this.shadowRoot.querySelector('#status').selected ? 'active': 'inactive';
```

Comment:
Settings the status as `inactive` will retrieve `inactive` users only, but not users with the statuses of `deleted` or `before-verification`. Please refer to the manager's implementation:
https://github.com/lablup/backend.ai-manager/blob/2e3824e0a8eebf9165c44e0fa616fdc4c2ed7c26/src/ai/backend/manager/models/user.py#L359-L363

Is this intended behavior? If not, keeping `is_active` instead of `status` might be proper.

Reply:
I agree with your opinion. I changed the field to `is_active` instead of using `status`.